web/controller/components/home/ControllerHome.js

```diff
@@ -96,147 +94,129 @@ export function ControllerHome() {
     setControlOnFocus(location.state?.controlOnFocus);
   }, []);
 
-  return [
-    <Header key={0} />,
-    <Container
-      key={1}
-      className={`${classes.container} ${classes.whiteSection}`}
-      maxWidth="xl"
-    >
-      <ControllerControlDrawer
-        controlId={controlOnFocus?.id}
-        controlType={controlOnFocus?.type}
-        onClose={() => {
-          _loadControls();
-          setControlOnFocus(null);
-        }}
-      />
-      <ControllerControlNewNoLic
-        isOpen={openNewNoLic}
-        onClose={() => setOpenNewNoLic(false)}
-        setControlOnFocus={setControlOnFocus}
-      />
-      <Typography
-        variant="h4"
-        component="h1"
-        key={1}
-        className={classes.titleHello}
+  return (
+    <>
+      <Header />
+      <Container
+        className={`${classes.container} ${classes.whiteSection}`}
+        maxWidth="xl"
       >
-        Bonjour, {controllerUserInfo.firstName}
-      </Typography>
+        <ControllerControlDrawer
+          controlId={controlOnFocus?.id}
+          controlType={controlOnFocus?.type}
+          onClose={() => {
+            _loadControls();
+            setControlOnFocus(null);
+          }}
+        />
+        <ControllerControlNewNoLic
+          isOpen={openNewNoLic}
+          onClose={() => setOpenNewNoLic(false)}
+          setControlOnFocus={setControlOnFocus}
+        />
+        <Typography
+          variant="h4"
+          component="h1"
+          key={1}
```

Comment:
:green_circle: Au lieu d'utiliser `key={1}` utiliser un texte qui a du sens (et qui ne changera pas si on ajoute une `Typography` au dessus) 
```suggestion
          key="welcome-message"
```
À faire pour le composant suivant également.

Reply:
Oui tu as raison, autant la retirer.